pom.xml

```diff
@@ -50,7 +50,7 @@
         <revision>1.34.6</revision>
         <changelist>-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/github-plugin</gitHubRepo>
-        <jenkins.version>2.346.1</jenkins.version>
+        <jenkins.version>2.357</jenkins.version>
```

Comment:
or prep for LTS
```suggestion
        <jenkins.version>2.361</jenkins.version>
```

Reply:
The stated purpose of this PR is to adapt to `instance-identity` changes, not to prep for LTS.